banks/urls.py

```diff
@@ -10,4 +23,16 @@
     path('account/<int:account_id>/update_balance/', UpdateAccountBalanceView.as_view(), name='update_balance'),
     path('account/<int:account_id>/delete/', DeleteAccountView.as_view(), name='delete_account'),
     path('account/create/', CreateAccountView.as_view(), name='create_account'),
+    path('banks/api/', BankListApiView.as_view(), name='api_view_banks'),
+    path('banks/api/viewset/', BankViewSet.as_view({'get': 'list'}), name='viewset_banks'),
+    path('banks/api/generic/', BankListGenericView.as_view(), name='generic_view_banks'),
+    path('api/login/', obtain_auth_token, name='api_login'),
+    path('accounts/api/', UserAccountApiView.as_view(), name='user_accounts'),
+    path('banks/api/create/', CreateBankApiView.as_view(), name='create_bank'),
+    path('accounts/api/create/', CreateAccountApiView.as_view(), name='create_account'),
+    path('accounts/api/update/<int:pk>/', UpdateAccountApiView.as_view(), name='update_account'),
+    path('accounts/api/detail/<int:pk>/', AccountDetailAPIView.as_view(), name='detail_account'),
+    path('accounts/api/delete/<int:pk>/', DeleteAccountApiView.as_view(), name='delete_account'),
+    path('accounts/api/detail-update-delete/<int:pk>/', AccountDetailUpdateDeleteAPIView.as_view(),
+         name='detail_update_delete_account'),
```

Comment:
theory: lets study this article and update url names accordingly
https://www.theserverside.com/video/Top-REST-API-URL-naming-convention-standards#:~:text=REST%20API%20URL%20naming%20conventions&text=Only%20use%20lowercase%20letters%20in,methods%20to%20perform%20an%20operation.